scripts/generate_all_docs.sh

```diff
@@ -3,18 +3,18 @@ set -xeuo pipefail
 
 if [ "${1-}" = "" ]
 then
-    echo >&2 "Usage: $0 duckdb_dir_with_debug_duckdb"
+    echo >&2 "Usage: $0 duckdb_dir_with_release_duckdb"
     exit 1
 fi
 
 DUCKDB=$1;
 echo "Generating docs using duckdb source in $DUCKDB"
 
-python3 ./scripts/generate_config_docs.py $DUCKDB/build/debug/duckdb
+python3 ./scripts/generate_config_docs.py $DUCKDB/build/release/duckdb
```

Comment:
If you're going to change these, you need to update the workflows that populate them, else the autoupdate will break

Reply:
Good point, thanks! Fix is incoming/